### PR TITLE
Feature/api implementation

### PR DESCRIPTION
**Pull request by lschnell8**
PR Template:

Is this a fix or a feature?
- [ ] Bug fix
- [x] Feature
- [ ] Refactor
 
Describe the feature - fix - refactor:
Wrote api fetch calls and implemented api data

How was this tested?
- [ ] open index.html
- [x] localhost
- [x] terminal

Where should the reviewer start - test:
scripts.js

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-// Your fetch requests will live here!
-
-
-console.log('I will be a fetch request!')
\ No newline at end of file
+export const fetchApiData = (type) => {
```
Comment (StevieAmb) [inline src/apiCalls.js:1]: This is a good catch with the interpolation, and making it dynamic.

File: src/scripts.js [hunk 2]
```diff
@@ -51,32 +52,90 @@ const totalCost = document.getElementById('totalCost')
...
 let newUser;
 let currentRecipe; 
+let groceryStore;
+let randomUser;
+let randomRecipe;
 
+//ON PAGE LOAD
 
+//FUNCTIONS
+const loadPage = () => {
```
Comment (StevieAmb) [inline src/scripts.js:68]: This is cool seeing how the fetch API works.

File: src/scripts.js [hunk 3]
```diff
@@ -40,6 +40,7 @@ const recipeResultsView = document.getElementById('recipeResultsView');
 const recipeInfoView = document.getElementById('recipeInfoView');
```
Comment (Alynn022) [inline src/scripts.js:40]: Can you comment back in this element?

**Review by StevieAmb — APPROVED** [review]

File: src/scripts.js [hunk 4]
```diff
@@ -40,6 +40,7 @@ const recipeResultsView = document.getElementById('recipeResultsView');
...
 const allRecipesView = document.getElementById('allRecipesView');
 const favoriteRecipesView = document.getElementById('favoriteRecipesView');
+const recipesToCookView = document.getElementById('recipesToCookView');
 // const filteredFavoriteRecipesView = document.getElementById('userFaveRecipeFilteredView');
```
Comment (Alynn022) [inline src/scripts.js:44]: We can delete this code comment

File: src/scripts.js [hunk 5]
```diff
@@ -51,32 +52,90 @@ const totalCost = document.getElementById('totalCost')
 const favRecipeCards = document.getElementById('favRecipeCards')
 // const randomRecipeImage = document.getElementById('randomRecipeImage');
```
Comment (Alynn022) [inline src/scripts.js:53]: This code comment can be deleted as well

**Review by Alynn022 — CHANGES_REQUESTED** [review]

**Pull request merged**